package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "gaia-toolbar",
+  "description": "A toolbar comopnent that is groups buttons and other controls together",
+  "version": "0.0.1",
+  "homepage": "https://github.com/gaia-components/gaia-toolbar",
+  "license": "MIT",
+  "main": "gaia-toolbar.js",
+  "devDependencies": {
+    "bower": "^1.4.1",
+    "chai": "3.4.1",
+    "jshint": "^2.8.0",
+    "karma": "0.13.15",
+    "karma-firefox-launcher": "0.1.7",
+    "karma-mocha": "^0.2.0",
+    "karma-sinon-chai": "1.1.0",
+    "marionette-client": "1.9.2",
+    "marionette-firefox-host": "1.0.4",
+    "marionette-helper": "0.3.2",
+    "marionette-js-runner": "1.1.3",
+    "mocha": "2.3.4",
+    "mozilla-download": "1.1.1",
+    "test-utils": "gaia-components/test-utils"
+  },
+  "scripts": {
+    "preinstall": "npm i socket-retry-connect json-wire-protocol sockit-to-me && ln -s node_modules/socket-retry-connect/ socket-retry-connect && ln -s node_modules/json-wire-protocol/ json-wire-protocol && ln -s node_modules/sockit-to-me/ sockit-to-me",
```

Comment:
npm install marionette-client doesn't work because its dependencies are expressed locally. I don't have any other idea of how to make it actually work except for creating a symlinks for these dependencies that will resolve the relative paths and then removing them after install is done.